src/Magnum/Platform/GlfwApplication.h

```diff
@@ -436,6 +437,33 @@ class GlfwApplication {
         /** @copydoc Sdl2Application::redraw() */
         void redraw() { _flags |= Flag::Redraw; }
 
+    public:
+        enum class Cursor : UnsignedInt {
+            Arrow,      /**< Arrow */
+            IBeam,      /**< I-Beam */
```

Comment:
I had to think twice about what this means... CSS has it simply as `text`, let's use that instead? :)